flow-typed/redux/appState.js

```diff
@@ -7,6 +7,19 @@ import {
   type Channel,
 } from 'redux-saga'
 
+declare type Router5Route = {
+  +name: string,
+  +params: {
+    [key: string]: any,
+  },
+  +path: string,
+}
```

Comment:
Please, add `TODO` for `meta` property pass

Reply:
Fixed in related PR #719